lib/rdoc/parser/ruby.rb

```diff
@@ -164,10 +165,20 @@ class RDoc::Parser::Ruby < RDoc::Parser
   def initialize(top_level, file_name, content, options, stats)
     super
 
+    if /\t/ =~ content then
+      tab_width = @options.tab_width
+      content = content.split(/\n/).map do |line|
+        1 while line.gsub!(/\t+/) {
+          ' ' * (tab_width*$&.length - $`.length % tab_width)
+        }  && $~
+        line
+      end.join("\n")
+    end
```

Comment:
Too bad [expand_tab](https://github.com/ruby/rdoc/blob/4b2fab86d907f5f0115e50d322238d431172404e/lib/rdoc/text.rb#L68) doesn't support `@options.tab_width`

Reply:
I [filed an issue about it](https://github.com/ruby/rdoc/issues/513).